TheaterDatabase/TestData/CleanupDatabase.sql

```diff
@@ -1,77 +0,0 @@
-USE TheaterDatabase
```

Comment:
why this script removed?

Reply:
It's okay, I moved cleaning database and filling into one script